hack/deploy-hub-local/ansible/setup.yaml

```diff
@@ -79,3 +79,44 @@
       service:
         name: sshd
         state: restarted
+
+    - name: Add Slack notifications
```

Comment:
Not sure about this task regarding recent discussions, probably we'll have to leave it out of the playbooks until we figure the way we want to be notified about the server statuses.